src/main/java/com/example/spring/models/CartItem.java

```diff
@@ -0,0 +1,36 @@
+package com.example.spring.models;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
+
+@Entity
+@Table(name = "cart_items")
+@Getter
+@Setter
+@SQLDelete(sql = "UPDATE cart_items SET is_deleted = TRUE WHERE id = ?")
+@SQLRestriction(value = "is_deleted = FALSE")
+public class CartItem {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
```

Comment:
```suggestion
    @ManyToOne(fetch = FetchType.LAZY, optional = false)
```

Reply:
Why do I need use fetch type Lazy, if I have many to one relation?

Reply:
This is a rule of good code that shows that you are a developer who cares about the performance of the application in the early stages of development